backend/config/curation-api.yml

```diff
@@ -1023,11 +1023,11 @@ components:
       required: true
       schema:
         $ref: "#/components/schemas/collection_id"
-    query_dataset_id:
+    path_dataset_id:
       description: The uuid of the Dataset.
-      in: query
+      in: path
       name: dataset_id
-      required: false
+      required: true
```

Comment:
Since everything is a uuid now, I think we should consider using the [pattern validator](https://swagger.io/docs/specification/data-models/data-types/#pattern), for both collection_id and dataset_id

Reply:
connexion doesn't support this, so there will be 0 enforcement unless we implement it. Also, this value is never generated by the user, so there is no reason or them to know the format. Lastly, if we validate the shape of every id, then we'll have to update a bunch of tests.

Reply:
Hm, I don't agree. They could think a given dataset simply doesn't exist, when in reality they entered the value wrong or accidentally truncated it with a keystroke when they weren't paying attention. It'd be useful for them to know that the id they entered isn't actually a uuid in that situation. Admittedly it's something we could handle in server code (a more detailed error message if the id they provide isn't actually a uuid). But yes, I agree it would incur extra work in the form of modifying all test id's that we use. But we don't have to do this right now, so I'm okay with ignoring it.

I didn't know connexion doesn't support the built-in `pattern` keyword though 😢